packages/api-db-mongodb/src/db/session.ts

```diff
@@ -99,6 +99,32 @@ export class MongoDBSessionAdapter implements DBSessionAdapter {
     return null
   }
 
+  async extendUserSessionByToken(token: string): Promise<OptionalUserSession> {
```

Comment:
I think it's so helpful to keep users logged-in depending on specific condition(s) 👍